### PR TITLE
define macros for print levels and avoid direct use of fmt::printf

### PR DESCRIPTION
**Pull request by eyvallah**

### CODE REVIEW

File: src/celestia/oggtheoracapture.cpp [hunk 1]
```diff
@@ -309,13 +309,13 @@ bool OggTheoraCapture::start(const std::string& filename,
...
     yuv.uv_stride=video_x/2;
 
-    fmt::printf(_("OggTheoraCapture::start() - Theora video: %s %.2f(%d/%d) fps quality %d %dx%d offset (%dx%d)\n"),
-                filename.c_str(),
-                (double)video_hzn/(double)video_hzd,
-                video_hzn,video_hzd,
-                video_q,
-                video_x,video_y,
-                frame_x_offset,frame_y_offset);
+    DPRINTF(PRINT_LEVEL_VERBOSE,
```
Comment (pirogronian) [inline src/celestia/oggtheoracapture.cpp]: I think _(...) may stay here. It's better to have localized messages.

File: src/celestia/oggtheoracapture.cpp [hunk 2]
```diff
@@ -424,7 +424,7 @@ void OggTheoraCapture::cleanup()
 
     if(outfile)
     {
-        fmt::printf(_("OggTheoraCapture::cleanup() - wrote %d frames\n"), video_frame_count);
+        DPRINTF(PRINT_LEVEL_VERBOSE, "OggTheoraCapture::cleanup() - wrote %d frames\n", video_frame_count);
```
Comment (pirogronian) [inline src/celestia/oggtheoracapture.cpp]: Same here.

File: src/celutil/debug.h [hunk 3]
```diff
@@ -20,6 +20,10 @@
 #undef DPRINTF // OSX has DPRINTF
 #endif // DPRINTF
 
+#define PRINT_LEVEL_ERROR       0
```
Comment (pirogronian) [inline src/celutil/debug.h]: Verbosity levels are usually ordered from Error to Debug, ie Debug is more verbise than normal info and normal info is mire verbose than error. In more sophisticated systems I saw following levels:
error
warning
info
verbose
debug

File: src/cel3ds/3dsread.cpp [hunk 4]
```diff
@@ -620,18 +620,18 @@ M3DScene* Read3DSFile(ifstream& in)
     unsigned short chunkType = readUshort(in);
     if (chunkType != M3DCHUNK_MAGIC)
     {
-        DPRINTF(0, "Read3DSFile: Wrong magic number in header\n");
+        DPRINTF(PRINT_LEVEL_ERROR, "Read3DSFile: Wrong magic number in header\n");
```
Comment (375gnu) [inline src/cel3ds/3dsread.cpp]: Please rename PRINT_LEVEL_ to LOG_LEVEL_

**Comment by 375gnu** [comment]
on the other hand why not to use https://github.com/gabime/spdlog?

**Comment by eyvallah** [comment]
> on the other hand why not to use https://github.com/gabime/spdlog?

let's leave it to another PR :)

**Comment by eyvallah** [comment]
@pirogronian pls approve

**Pull request merged**